internal/provider/worker_test.go

```diff
@@ -118,7 +118,7 @@ func TestWorkerControllerLed(t *testing.T) {
 					resource.TestCheckResourceAttrSet("boundary_worker.controller_led", "controller_generated_activation_token"),
 				),
 			},
-			importStep("boundary_worker.controller_led"),
+			importStep("boundary_worker.controller_led", authorizedActions),
```

Comment:
Added ignore field for "authorized actions" because it the order of the list was different from what was expected. The actions were the same but the order changed so the test failed. Ignoring field in the test